docs/rules/no-debug.md

```diff
@@ -28,6 +28,12 @@ const { screen } = require('@testing-library/react');
 screen.debug();
 ```
 
+If you want to disallow the use of other debugging functions, you can configure what names this rule checks for with the `utilNames` option:
```

Comment:
The description of this rule must be updated to mention which debugging utils are reported by default.